data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -1117,9 +1124,11 @@ Please report the bug."
         name=moveto
         [filter]
             side=1
-            x=1-20
-            y=18-21
         [/filter]
+        [filter_location]
+            x,y=38,5
+            radius=5
+        [/filter_location]
```

Comment:
I couldn't get this one to trigger

Reply:
Strangely enough, it works perfectly fine for me.

Reply:
I retried, and it seems broken but in a different way - it triggers on the very first move by side 1, when Konrad/Li'sar steps off their keep.

Reply:
Huh, happens to me too.

Reply:
This should fix it.